frontend/pages/admin/campaigns/create.tsx

```diff
@@ -130,7 +125,7 @@ const CampaignCreate = () => {
       });
     } else {
       // Note: This has to be here otherwise the previous interest values may be shown
-      setInitialValues(defaultInitialValues);
+      setInitialValues(campaignDefaultInitialValues);
```

Comment:
I'm not able to submit the create campaign form using the create button (not from an existing interest), is this expected? Error: `interestId is a required field`

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/7360964/196017493-918f2284-415a-41f5-9d09-e0343bcc4ddf.png">